client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -379,7 +381,8 @@ void main(void)
 #endif
 
 	color = base.rgb;
-	vec4 col = vec4(color.rgb * varColor.rgb, 1.0);
+
+	vec4 col = vec4(color.rgb * (varColor.rgb + ambientLight), 1.0);
```

Comment:
Please do `min(varColor.rgb + ambientLight, 1.0)` so that proper fullbright is possible by setting `ambientLight` to \#FFFFFF. Same for the object shader.